### PR TITLE
Added 3 new uplink items and a new item category.

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added instant bear, banhammer and instakill gun to traitor uplink.
add: A new category of items has been added called "Collaborative Gear". Work together with your fellow agents to obtain powerful items!
tweak: Romerol has been moved to "Collaborative Gear"
/:cl:
Just simple additions to the uplink. The instant bear is useful for when you are chased, the banhammer is a fun toy, and you require the aid of 4 other traitors along with yourself to acquire it. If you somehow managed the gargantuan task of convincing people to give you TC without having the instakill gun be theirs, then you probably deserve the right to kill everyone on the station. PopNotes' suggestion to add collaborative gear.

### CODE REVIEW

**Comment by ghost** [comment]
i uh, really don't know about the instant kill gun

i really don't think co-operation like that will ever happen but the idea of an instant kill weapon existing is bad.

but i do also really appreciate the fact there will only be 1 gun and ultimately, only 1 traitor can have it.

**Comment by Pyko1** [comment]
So basically its nuke op leader swindling all the tc or someone exploiting.

Do we really want to encourage either one?

**Comment by Arzion5** [comment]
I've already added an exclusion from nuke ops. The items under the "Collaborative" category will not appear in Nuclear Emergency, and also cannot be discounted unless specified otherwise.

**Comment by tortellinitony** [comment]
Seems like a pointless addition

**Comment by DinduStuffin** [comment]
better idea, retool this to work like some of the weapons in Metal Gear Solid: Peace Walker.
The Human Slingshot is an excellent example. Get the traitors to pool their TC together, and buy the slingshot kit. Two people get posts, one gets a rubber thing, and one human becomes the ammo.

Instant kill weapons like the proposed banhammer basically encourage people to go around killing other antags as fast as they can so they can murderbone as much as they can. I'd rather have a band of retards using gimmicky human slingshots to kill each other than people fighting over TC to get a super broken weapon.

I love the idea of collaboration focused items and gimmicks though. That's a damn good idea imo.

**Comment by Arzion5** [comment]
Actually, traitors do not know the uplink codes of other players. They will lose the potential amount of TC if they kill their fellow agents. The banhammer is just the toy version, which is why it's listed under the "Badass" category. I wish I knew coding well enough to add a human slingshot though

**Comment by ghost** [comment]
@DinduStuffin Banhammer isn't instant kill. It just gives a fake "U HAVE BEEN BANNED FOR NO RAISINS BY USER" it's a toy item.

**Comment by j-mie** [comment]
[11:51 PM] Jambread: @Arzion ATOMIC PRS
[11:51 PM] Jambread: 1 change per a PR
[11:51 PM] Banthebantz: nukes
[11:51 PM] Jambread: don't edit the health of a rando mob and edit the uplink? wat
NEW MESSAGES
[11:51 PM] Banthebantz: makes it more awkward for admins to come to a decision yes

**Comment by GuyonBroadway** [comment]
Keep your pr changes limited to one subject at a time. Put the watcher hp nerf in a different pr please m

**Comment by tortellinitony** [comment]
I might be the only one, but I'm not a big fan of adding all of this pointless gimmicky trash to the traitor uplink

**Review by KayeArray — DISMISSED** [review]
provided it's been tested, the code looks fine

**Comment by ghost** [comment]
fixed merge conflicts

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -226,6 +231,34 @@
...
+	exclude_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/colab/instakill
+	name = "Instakill Gun"
+	desc = "An extremely rare and advanced weapon that can flat out rip apart any being in one shot."
+	item = /obj/item/gun/energy/laser/instakill
+	cost = 100
+
+/datum/uplink_item/stealthy_weapons/romerol_kit
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/traitor)
```
Comment (ghost) [inline hippiestation/code/modules/uplink/uplink_item.dm:255]: ```category = "Collaborative Gear"```

**Review by ghost — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/uplink/uplink_item.dm:255]: He's overriding it to disable that version of the uplink item.

He redefines a colab based version below this.

Comment (ghost) [reply hippiestation/code/modules/uplink/uplink_item.dm:255]: oh

**Comment by Arzion5** [comment]
It's been too long and I have plans on adding just the collaborative gear. Closing, moving collaborative gear to new PR

**Pull request closed without merging**